integrations/phishing_analyzers/compose.yml

```diff
@@ -0,0 +1,39 @@
+# IMPORTANT: The version must match the version of docker-compose.yml
+---
+# All additional integrations should be added following this format only.
+
+services:
+  phishing_analyzers:
+    image: intelowlproject/intelowl_phishing_analyzers:${REACT_APP_INTELOWL_VERSION}
+    container_name: intelowl_phishing_analyzers
+    restart: unless-stopped
+    expose:
+      - "4005"
+      - "7007"  # selenium-wire proxy
+    env_file:
+      - env_file_integrations
+    volumes:
+      - generic_logs:/var/log/intel_owl
+    depends_on:
+      - uwsgi
+
+  chrome-webdriver:
+    image: selenium/node-chrome:latest
+    shm_size: 2gb
```

Comment:
This value is suggested in Docker Selenium [doc](https://github.com/SeleniumHQ/docker-selenium?tab=readme-ov-file#--shm-size2g) to prevent browsers from crashing inside of containers.

Reply:
ok please add this information as a comment for future reference